### PR TITLE
Updated CI info in contributing docs

### PR DESCRIPTION
**Pull request by 2-5**
# Description

I tried following the `CONTRIBUTING.md` and `js/README.md` instructions and found that it still mentions `circleci` which apparently was removed in January.

So I updated the docs with a description of the current GitHub actions.

# Checklist:

- [x] I have performed a self-review of my own code
- [ ] I have added a short summary of my change to the CHANGELOG.md
- [x] My code follows the style guidelines of this project
- [ ] I have commented my code in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Review by pngwn — APPROVED** [review]
Thanks @2-5!

File: js/README.md [hunk 1]
```diff
@@ -90,6 +90,19 @@ pnpm ts:check
 
 Currently the following checks are run in CI:
 
+### static checks
+
 - Format check (`pnpm format:check`)
+- Build css (`pnpm css`)
+- Build client (`pnpm build`)
```
Comment (freddyaboulton) [inline js/README.md:97]: Is the build client step needed here @pngwn ?

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -91,10 +91,23 @@ bash scripts/format_backend.sh
 bash scripts/format_frontend.sh
 ```
 
-You can run the circleci checks locally as well.
+## CI checks
+
+Currently the following checks are run in CI:
+
+```
```
Comment (freddyaboulton) [inline CONTRIBUTING.md:98]: This code block here is for the client. Can we add a sub-header here for "python client" .

The other subheader can be "gradio library" and its contents should be

```
bash scripts/lint_backend.sh
bash scripts/type_check_backend.sh
python -m pytest -m "not flaky" --ignore=client
python -m pytest -m "flaky" --ignore=client
```

Do you mind making the change @2-5 ?

File: js/README.md (hunk 1 above)
Comment (pngwn) [reply js/README.md:97]: Nah it happens as part of the build command I think. Will doublecheck.

Comment (pngwn) [reply js/README.md:97]: Yeah it does.

Comment (freddyaboulton) [reply js/README.md:97]: Word - thanks for checking!

File: CONTRIBUTING.md (hunk 2 above)
Comment (2-5) [reply CONTRIBUTING.md]: Ok, I have made the changes.

**Review by freddyaboulton — APPROVED** [review]
Thanks @2-5 ! Will merge after CI passes

**Pull request merged**